.github/workflows/release.yaml

```diff
@@ -6,142 +6,16 @@ on:
       - main
 
 jobs:
-  lib-check:
-    name: Check libraries
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-      - name: Check libs
-        uses: canonical/charming-actions/check-libraries@2.1.0
-        with:
-          credentials: "${{ secrets.CHARMHUB_TOKEN }}" # FIXME: current token will expire in 2023-07-04
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
This will be just lost.
I recommend to move it into ci.yaml, see mysql-operator.git as an example.

Please update to 2.2.2 version of charming-actions. It contains a lot of fixes!
Also consider to add canonical/charming-actions/release-libraries as your repo has libraries good to puclish automatically.

Reply:
Good catch! Should I put this in the CI (given it has secrets and so far the CI pipeline is secret free) or best in the release only?

Reply:
Also, we actually have some libs there that have not yet been published I believe (like kafka client). So I'd just hold this off for the time being if that is fine